packages/dso-toolkit/src/components/delete/delete.mixins.scss

```
@@ -17,6 +19,8 @@
 @mixin strikethrough($_value: line-through) {
   text-decoration: $_value;
 
+  --_dso-ozon-content-toggletip-text-decoration: underline #{$-value};
```

Moet het niet zijn:


```suggestion
  --_dso-ozon-content-toggletip-text-decoration: underline #{$_value};
```

Of begrijp ik niet wat `#{$-value}` doet?

---

Slordig.

Ik heb het gecorrigeerd.